src/Enzo/enzo_EnzoMethodPmDeposit.cpp

```diff
@@ -391,9 +490,16 @@ void EnzoMethodPmDeposit::compute ( Block * block) throw()
     int nzi=mz-gz-1;
     int i0 = 0;
     int i1 = 1;
-    enzo_float hxf = hx;
-    enzo_float hyf = hy;
-    enzo_float hzf = hz;
+
+    /* Stefan: Not sure if these next four lines are correct.
+       I include the factors of cosmo_a for consistency with 
+       previous version. Also, not sure why dtf is
+       initialised with the value of alpha_.
+     */
+    
+    enzo_float hxf = hx * cosmo_a;
+    enzo_float hyf = hy * cosmo_a;
+    enzo_float hzf = hz * cosmo_a;
```

Comment:
I think this cosmo_a factors should not be here, for the same reason discussed earlier.